src/components/UI/AvailabilityZonesLabel.js

```diff
@@ -86,16 +86,15 @@ function AvailabilityZonesLabel({
     isMaxReached && !isRadioButtons ? 'is-max-reached' : '';
   const classNames = `${letter} ${notCheckedClass} ${pointerClass} ${isMaxReachedClass}`;
 
+  const onClick = () => {
+    isMaxReached && !isChecked && !isRadioButtons
```

Comment:
I think the ternary should be on the first level like this

```js
const onClick = onToggleChecked ? () => { 
    isMaxReached && !isChecked && !isRadioButtons
      ? null
       : () => onToggleChecked(!isChecked, { title, letter, label }) } : undefined;
```

didn't test this though. Looks fine otherwise